go.mod

```diff
@@ -4,16 +4,16 @@ go 1.23.4
 
 require (
 	github.com/askervin/gofmbt v0.0.0-20250119175120-506d925f666f
-	github.com/containerd/nri v0.6.0
+	github.com/containerd/nri v0.9.1-0.20250530003506-6120e633d4ad
```

Comment:
Have you tested if this has some unwanted side-effect on an otherwise unmodified existing balloons or t-a policy in a cluster which runs an stock runtime using an older version of tagged NRI ?

Reply:
@askervin Thanks for the extra tests you ran. Perhaps we should then be brave enough and merge this. We can deal with any problems if it turns out we run into something unexpected.